modules/tinymce/CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Improved
 - Transparent elements, like anchors, can now contain block elements. #TINY-9172
+- Tests for insert content and get content events. #TINY-9143
```

Comment:
Remove this since tests are not something that affects the release. So a integrator would not see anything different in the usage of tinymce based on this change.

- [ ] Remove changelog for tests change

Reply:
Removed this line [here](https://github.com/tinymce/tinymce/pull/8215/commits/014f9e2f3f9121a3f0f12c70c4bfd5183a223999)